Source/Android/jni/AndroidCommon/IDCache.cpp

```diff
@@ -343,12 +343,12 @@ jfieldID GetRiivolutionPatchesPointer()
 
 jclass GetWiiUpdateCallbackClass()
 {
-    return s_wii_update_cb_class;
+  return s_wii_update_cb_class;
```

Comment:
Please address review comments and lint corrections in the commit the problem was introduced in. (Also applies to the changes below.)